src/hiccup.rs

```diff
@@ -126,7 +137,22 @@ pub fn render_by_depth(element: &Value, depth: usize) -> String {
                     if key.eq("checked") {
                         output = format!("{} {}", output, key);
                     } else {
-                        output = format!("{} {}=\"{}\"", output, key, value.as_str().unwrap());
+                        output = format!(
+                            "{} {}=\"{}\"",
+                            output,
+                            key,
+                            match value {
+                                Value::String(s) => s.as_str().to_string(),
```

Comment:
Why `s.as_str().to_string()`? Don't we just want `s`?

Reply:
Yes you're right. We don't need those calls to `.as_str().to_string()`.

Regarding whether it's a bug or a feature request: Sure, you can call it whatever you like :-) From my point of view I would just like this library to support the same inputs as the python version :-)